src/lib/wizard/components/Deploy.svelte

```diff
@@ -378,4 +387,16 @@
       <Button label={"View Deployment"} onClick={() => window.open(deploymentUrl, "_blank")} type="secondary" />
     {/if}
   {/if}
+
+  {#if !isCompiling}
+  <div class="alert alert-success d-flex align-items-center mt-2">
+    <div class="flex flex-row items-center gap-2">
+      <i class={`fa fa-lightbulb-o text-lime-600`}></i>
+      <div class="text-xs text-gray-600">
+        <p>Ensure you have an Explorer API Key set in your <u><a href='https://defender.openzeppelin.com/#/deploy' target='_blank'>Deploy Environment</a></u> for the current network to allow the contract to be verified automatically.</p>
```

Comment:
If at this point we have network selected we could redirect to the related deploy environment (like `deploy/environment/test`)